sys/include/net/gnrc/netdev.h

```diff
@@ -160,6 +160,9 @@ typedef struct gnrc_netdev {
     gnrc_lwmac_t lwmac;
 #endif
 
+#ifdef MODULE_GNRC_GOMACH
+    gnrc_gomach_t gomach;
+#endif
```

Comment:
This struct doesn't need updating (in fact this file should have been removed in #7925, see #8180).